instrumentTest/crypto/src/com/facebook/crypto/NativeGCMCipherInputStreamTest.java

```diff
@@ -217,4 +217,24 @@ public void testCompatibleWithBouncyCastle() throws Exception {
     inputStream.close();
     assertTrue(CryptoTestUtils.DECRYPTED_DATA_IS_DIFFERENT, Arrays.equals(mData, decryptedData));
   }
+
+  /**
+   * Test invoking close multiple times does not throw exceptions.
+   * NativeGCMCipherInputStream extends java.io.InputStream. InputStream
+   * implements Closeable. Closeable requires that if the stream is already
+   * closed then invoking this method has no effect.
+   */
```

Comment:
I'd get rid of the comments too. The code is fairly obvious and there's no need to get into implementation details.